storage/bbolt/dedup.go

```diff
@@ -196,21 +196,31 @@ func btoi(b []byte) uint64 {
 
 // vtob concatenates an index and timestamp values into a byte array.
 func vtob(idx uint64, timestamp uint64) ([]byte, error) {
-	i := itob(idx)
-	if len(i) != 8 {
-		return nil, fmt.Errorf("input error, idx should be %d bytes long, got %d", 8, len(i))
+	b := make([]byte, 16)
```

Comment:
This looks like a bug - I suspect you'll be returning 32 byte slices here, and when unmarshaling in`btov` you'll find `timestamp` and `idx` are both `0x0`

Reply:
Oh yeah it is, well spotted, thanks. ",0" was here and then removed during reformatting, I ran manual tests with a test instance but did not spot this. Fixed now. And yes - this really needs to be tested programatically. Additional SCTFE tests are currently blocked on simplifying the architecture, and getting rid of mocks. But it should be possible to test this package without waiting for this to be done. Will follow up.